extension/docs/CreditCardIntegration.md

```diff
@@ -0,0 +1,67 @@
+## Credit card payment
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [Credit card](#credit-card)
+- [Credit card with 3D Secure](#credit-card-with-3d-secure)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Adyen documentation: https://docs.adyen.com/developers/payment-methods/cards (notice: Recurring card payments is not supported, manual capture is not supported)
```

Comment:
The following features are not supported:   
- Recurring card payments
- Manual capture